src/bootstrap/mk/Makefile.in

```diff
@@ -115,13 +115,7 @@ ci-msvc: ci-msvc-py ci-msvc-ps1
 
 ## MingW native builders
 
-# Set of tests that should represent half of the time of the test suite.
-# Used to split tests across multiple CI runners.
```

Comment:
Nit: I believe this comment is still accurate (see below for `ci-mingw-x` and `ci-mingw-bootstrap`. cc @marcoieni since I could be mistaken.

Reply:
yes, those two lines are still accurate for `ci-mingw-x` and `ci-mingw-bootstrap` 👍
Can you bring them back?
Thanks for noticing!